README.md

```diff
@@ -1,20 +1,60 @@
 # design-pattern-taiko
 
-# Install Dependencies
+## Requirements
+* npm
+* docker
+* Code Editor: visual studio code
+
+## Install Dependencies
 `npm install`
 
-# Bring up the wordpress app in local
+## Bring up the wordpress app in local
 `docker-compose -f docker-compose.yml up`
```

Comment:
I see the docker-compose file is missing here.

Reply:
@SrinivasanTarget I have added the docker-compose to master